chpl/chpl-resources/src/main/resources/environment.properties

```diff
@@ -216,3 +216,6 @@ realWorldTestingCriteriaKeys=criterion.170_315_b_1_old,\
 auditDataFilePath=SECRET
 auditDataRetentionPolicyInMonths=2
 ####################################
+
+
+useUpdatedCertifiedProductDetailsManager=false
```

Comment:
I don't think this is used anymore, is it? Can you also remove from the DEV bamboo plan?